input/about/diversity-and-inclusion.md

```diff
@@ -0,0 +1,20 @@
+# Promoting and Studying Diversity and Inclusion in .NET Open Source
+
+An important goal of the .NET Foundation is to promote diversty and inclusion within the .NET community and OSS ecosystem. To that end, we've set upon the following goals:
+
+## Short Term
```

Comment:
How long is Short, Medium and Long Term? Maybe we should at least say "In the next 6 months" for short term? What committee(s) are driving these?